ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -507,6 +507,7 @@ def test_numpy_negative_binomial(
     fn_tree,
     backend_fw,
     on_device,
+
```

Comment:
we can remove this line.

Reply:
But I am working on standard_exponential not on negative_binomial

Reply:
Nit: Yes, but some unnecessary changes have been made here. It's okay it's just nit.

Reply:
What should I do now?